portfolio/src/main/webapp/index.html

```diff
@@ -9,11 +9,17 @@
   <body>
   <div class ="sidebar">
     <a class="active" href="#About Me">Home</a>
-    <a href="#contact">Contact Info</a>
-    <a href="#resume">Resume</a>
-  </div>
-    <div id="content">
-      <h1> Nicole Avila's</h1>
+    <a class="resume" href="https://docs.google.com/document/d/1OMKg2K1GW6W3wr_lU_Z7RtVczClg4CL6yqtsI2lSlrw/edit?usp=sharing">Resume</a>
+    <p> Phone Number: 919-923-7482 <br>
```

Comment:
Nit: Generally i think it's better to move the <br> to the next line and end it with a closing </br>